assets/js/src/newsletters/newsletters.jsx

```diff
@@ -50,64 +47,63 @@ if (container) {
   let routes = [
     /* Listings */
     {
-      path: 'standard(/)**',
-      params: { tab: 'standard' },
+      path: '/standard(/)**',
       component: NewsletterListStandard,
     },
     {
-      path: 'welcome(/)**',
+      path: '/welcome(/)**',
       component: NewsletterListWelcome,
     },
     {
-      path: 'notification/history/:parent_id(/)**',
+      path: '/notification/history/:parent_id(/)**',
       component: NewsletterListNotificationHistory,
     },
     {
-      path: 'notification(/)**',
+      path: '/notification(/)**',
       component: NewsletterListNotification,
     },
-    /* Newsletter: type selection */
-    {
-      path: 'new',
-      component: NewsletterTypes,
-    },
     /* New newsletter: types */
     {
-      path: 'new/standard',
+      path: '/new/standard',
       component: NewsletterTypeStandard,
     },
     {
-      path: 'new/notification',
+      path: '/new/notification',
       component: NewsletterTypeNotification,
     },
     {
-      path: 'new/welcome',
+      path: '/new/welcome',
       component: NewsletterTypeWelcome,
     },
+    /* Newsletter: type selection */
+    {
+      path: '/new',
+      component: NewsletterTypes,
+    },
     /* Template selection */
     {
       name: 'template',
-      path: 'template/:id',
+      path: '/template/:id',
       component: NewsletterTemplates,
     },
     /* congratulate */
     {
-      path: 'send/congratulate/:id',
+      path: '/send/congratulate/:id',
       component: NewsletterCongratulate,
     },
     /* Sending options */
     {
-      path: 'send/:id',
+      path: '/send/:id',
       component: NewsletterSend,
     },
   ];
 
   routes = Hooks.applyFilters('mailpoet_newsletters_before_router', [...routes, ...getAutomaticEmailsRoutes()]);
 
   window.mailpoet_listing = ReactDOM.render(( // eslint-disable-line react/no-render-return-value
-    <Router history={history}>
-      <Route path="/" component={App}>
-        <IndexRedirect to="standard" />
+    <HashRouter>
+      <Switch>
+        <Route exact path="/" render={() => <Redirect to="/standard" />} />
         {routes.map(route => (
           <Route
```

Comment:
Since we are passing a `params` prop to the `Route`, the rendered component will receive two `params` props: `params` and `match.params`? What is the difference between these two? should we rename the `params` prop to avoid confusion?

Reply:
I didn't check previously, but it seems that the `params` argument is not used and can be deleted.